radiography/src/main/java/radiography/Radiography.kt

```diff
@@ -15,17 +11,15 @@ import java.util.concurrent.TimeUnit.SECONDS
 
 /**
  * Utility class to scan through a view hierarchy and pretty print it to a [String].
- * Call [scan] or [View.scan].
+ * Call [scan] or [android.view.View.scan].
```

Comment:
Nit: the kotlin formatter should keep the import even if it's only used by kdoc.